fulltext/backends/__gz.py

```diff
@@ -33,6 +34,10 @@ def handle_fobj(self, path_or_file):
                 backend = backend_from_fname(orig_name)
             else:
                 backend = backend_from_fobj(f)
-            return get(f, backend=backend)
+            # This sucks but apparently some backends are not able to
```

Comment:
Is it possible to maintain a list of the backends that require this and only do it for them?

Reply:
I addressed this in 67caaf2cbc02d722b0d7be1f1d724212bb6c8de2 which tries using the file object first and if it fails it copies it on disk and retries.